Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml

```diff
@@ -3039,6 +3051,140 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+  "121":
+    id: "121"
+    taskid: 6c0ebe20-21a9-4e61-87e5-d4bdc630a92d
+    type: title
+    task:
+      id: 6c0ebe20-21a9-4e61-87e5-d4bdc630a92d
+      version: -1
+      name: Analysis Stage
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "123"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 615
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "123":
```

Comment:
Does there need to be a `Set alert summary grid field` task for the Analysis report to `asmalertsummary` until the final report is created or is that not useful data to store?

Reply:
We originally tried but the EV2 UI will only pic up the first item in there.  Andrew said he would create a follow up item for it (UI change).